contracts/network/BancorNetwork.sol

```diff
@@ -64,6 +64,9 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
     error DepositingDisabled();
     error NativeTokenAmountMismatch();
     error InsufficientFlashLoanReturn();
+    error PoolTradingShouldBeDisabled();
+    error PoolNotValid();
```

Comment:
error InvalidPool?

Reply:
removed as this error is not used anywhere - we check if pool is valid when retrieving the pool collection